backend/src/inngest/services.ts

```
@@ -0,0 +1,732 @@
+        break;
+      case "x_api_key":
+        headers[fieldName || "x-api-key"] = config.apiKey;
+        break;
+      case "query_param":
+        url = appendQueryParam(url, fieldName || "key", config.apiKey);
+        break;
+      case "body_field":
+        if (body === null || typeof body !== "object" || Array.isArray(body)) {
+          throw new Error("Request template must resolve to an object to inject the API key into the body");
+        }
+        const bodyField = fieldName || "api_key";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)